### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by p2227**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [ ] Make sure that you propose PR to right branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

### CODE REVIEW

**Comment by p2227** [comment]
https://github.com/react-component/dialog/blob/f10a2abbd82af40e9cc664bcb5e18f4e2ec30810/src/Modal.tsx#L53

File: components/modal/index.zh-CN.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ title: Modal
 | okText     | 确认按钮文字    | string           | 确定       |
 | okType     | 确认按钮类型     | string           | primary   |
 | cancelText | 取消按钮文字    | string           | 取消       |
-| maskClosable | 点击蒙层是否允许关闭 | Boolean   | `false`    |
+| maskClosable | 点击蒙层是否允许关闭 | Boolean   | `true`    |
```
Comment (benjycui) [inline components/modal/index.zh-CN.md]: 这个不对吧。。 Modal.xx 本来就是 false 的啊 https://ant.design/components/modal-cn/

Comment (benjycui) [reply components/modal/index.zh-CN.md]: see: https://github.com/ant-design/ant-design/blob/master/components/modal/confirm.tsx#L23

Comment (benjycui) [reply components/modal/index.zh-CN.md]: Reverted 59193ef

Comment (afc163) [reply components/modal/index.zh-CN.md]: 看错了。

Comment (p2227) [reply components/modal/index.zh-CN.md]: 好吧，原来Modal和Modal.confirm是不一样的

**Pull request merged**